web/migrations/0053_goods_featured.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.6 on 2025-04-04 09:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("web", "0052_add_meme_slug"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)